CMakeLists.txt

```diff
@@ -31,8 +31,9 @@ set(QUAD_PRECISION  ON  CACHE BOOL "Enable QUAD_PRECISION (for certain grid metr
 set(REPRO           OFF CACHE BOOL "Enable REPRO mode")
 set(WW3             OFF CACHE BOOL "Enable WW3")
 set(S2S             OFF CACHE BOOL "Enable S2S")
+set(CDEPS           OFF CACHE BOOL "Enable CDEPS")
 set(JEDI_DRIVER     OFF CACHE BOOL "Enable JEDI as top level driver")
-set(DATM            OFF CACHE BOOL "Enable Data Atmosphere")
+set(CDES_DATM       OFF CACHE BOOL "Enable CDEPS Data Atmosphere")
```

Comment:
This should be CDEPS_DATM

Reply:
Yes. I agree. This is already done in following PR https://github.com/hafs-community/ufs-weather-model/pull/8